core/src/rpc/aggregator.rs

```diff
@@ -235,30 +235,10 @@ async fn create_nonce_streams(
     }))
     .await?;
 
-    // Get the first responses.
+    // Get the first responses from verifiers.
     let first_responses: Vec<clementine::NonceGenFirstResponse> =
         try_join_all(nonce_streams.iter_mut().map(|stream| async {
```

Comment:
```suggestion
        try_join_all(nonce_streams.iter_mut().map(parser::verifier::parse_nonce_gen_first_response)
```

not sure if it compiles but

Reply:
lets do this in another pr: out of context and this pr needs to be merged quickly